test/pylib/scylla_cluster.py

```diff
@@ -1776,6 +1777,45 @@ async def _server_get_process_status(self, request: aiohttp.web.Request) -> Opti
         server_id = ServerNum(int(request.match_info["server_id"]))
         return self.cluster.server_get_process_status(server_id)
 
+async def get_scylla_2025_1_executable() -> pathlib.Path:
+    async def run_process(cmd, **kwargs):
+        proc = await asyncio.create_subprocess_exec(*cmd, **kwargs)
+        await proc.communicate()
+        assert proc.returncode == 0
+
+    url = 'https://s3.amazonaws.com/downloads.scylladb.com/downloads/scylla/relocatable/scylladb-2025.1/scylla-2025.1.0~rc3-0.20250223.aa5cb15166d3.x86_64.tar.gz'
```

Comment:
`https://downloads....` should work just as well and is more future-proof.

Reply:
Should we test debug mode against debug mode of older versions? Seems symmetrical and we do want to see if we invoke undefined behavior in order versions.

Reply:
Will address this in the separate series for the upgrade test.